lib/galaxy/web/framework/webapp.py

```diff
@@ -668,7 +668,8 @@ def get_history( self, create=False ):
         """
         history = None
         if self.galaxy_session:
-            history = self.galaxy_session.current_history
```

Comment:
If I'm reading this right -- the framework code here is used by the toolshed, but something is actually calling session.current_history, causing errors?  Shouldn't the toolshed just always say "no, you don't have one of those"?